static/js/components/EducationDialog.js

```diff
@@ -23,11 +26,16 @@ export default class EducationDialog extends ProfileFormFields {
   }
   educationLevelLabels: Object;
 
-  static propTypes = {
-    open:           React.PropTypes.bool,
-    onClose:        React.PropTypes.func,
-    onSave:         React.PropTypes.func,
-    showLevelForm:  React.PropTypes.bool,
+  props: {
+    ui:                           UIState,
+    setEducationDialogVisibility: () => void,
+    setEducationDegreeLevel:      () => void,
+    setEducationDialogIndex:      () => void,
+    clearProfileEdit:             () => void,
+    saveProfile:                  BoundSaveProfile,
```

Comment:
addendum to the above: i think we should get rid of the `SetWorkHistoryEdit` and `SetEducationDegreeInclusions` as named types. they're each used to describe one function, then they're used in one place each to indicate that a prop is of that type. this pattern suggests a world where every single function we declare needs an accompanying type declaration that exists alongside it, whether other functions will ever share that type or not. i would really not want to live in that world